docs/design/v0/components/agent-scrutiny/editor.tsx

```
@@ -0,0 +1,203 @@
+      <div className="flex-1 overflow-auto">
+        <table className="w-full text-sm font-mono">
+          <tbody>
+            {lines.map((line, index) => (
+              <tr key={index} className="hover:bg-muted/30">
+                <td className="w-12 text-right pr-4 text-muted-foreground select-none border-r border-border sticky left-0 bg-background">
+                  {index + 1}
+                </td>
+                <td className="pl-4 pr-8">
+                  <pre
+                    className={cn("whitespace-pre text-foreground")}
+                    dangerouslySetInnerHTML={{
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)